### ISSUE TITLE
1.0b6 fails to load settings (1.0b5 works)

### ISSUE THREAD

**Issue by gnosek** [PSU-OIT-ARC/django-local-settings#4]
Upon running `./manage.py --help` I get the stack trace like below. After some digging in, it looks like the object in question is a three-element tuple. After changing it to an array, `manage.py` spins in a loop using 100% CPU (there's a two-element tuple in the same dict as well, but changing it to an array doesn't change much, still spins).

```
Traceback (most recent call last):
  File "./manage.py", line 25, in <module>
    execute_from_command_line(sys.argv)
  File ".../lib/python3.5/site-packages/django/core/management/__init__.py", line 363, in execute_from_command_line
    utility.execute()
  File ".../lib/python3.5/site-packages/django/core/management/__init__.py", line 307, in execute
Django>=1.11,<2.0
    settings.INSTALLED_APPS
  File ".../lib/python3.5/site-packages/django/conf/__init__.py", line 56, in __getattr__
    self._setup(name)
  File ".../lib/python3.5/site-packages/django/conf/__init__.py", line 41, in _setup
    self._wrapped = Settings(settings_module)
  File ".../lib/python3.5/site-packages/django/conf/__init__.py", line 110, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File ".../lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File ".../settings.py", line 202, in <module>
    globals().update(load_and_check_settings(globals()))
  File ".../lib/python3.5/site-packages/local_settings/__init__.py", line 80, in load_and_check_settings
    settings, success = loader.load_and_check(base_settings, prompt)
  File ".../lib/python3.5/site-packages/local_settings/loader.py", line 31, in load_and_check
    settings = self.load(base_settings)
  File ".../lib/python3.5/site-packages/local_settings/loader.py", line 101, in load
    self._interpolate_values(settings, settings)
  File ".../lib/python3.5/site-packages/local_settings/loader.py", line 122, in _interpolate_values
    obj, interpolated = self._interpolate_values_inner(obj, settings)
  File ".../lib/python3.5/site-packages/local_settings/loader.py", line 131, in _interpolate_values_inner
    obj[k], _interpolated = self._interpolate_values_inner(v, settings, _interpolated)
  File ".../lib/python3.5/site-packages/local_settings/loader.py", line 131, in _interpolate_values_inner
    obj[k], _interpolated = self._interpolate_values_inner(v, settings, _interpolated)
  File ".../lib/python3.5/site-packages/local_settings/loader.py", line 137, in _interpolate_values_inner
    self._interpolate_values_inner(v, settings, _interpolated) for v in obj)
ValueError: too many values to unpack (expected 2)
```

**Comment by wylee** [comment]
Thanks for the report. Would you be able to share a sanitized copy of your settings?

**Comment by wylee** [comment]
I was able to reproduce. Working on a fix now.

**Comment by wylee** [comment]
Fixed in a6c6c2685058abec53b6974d7c20f08fdea48f8c. Released in 1.0b7.
